cgi/generate_sample_import_file.pl

```diff
@@ -198,9 +198,15 @@
 }
 
 my $tagtype = 'categories';
-my @category_entries = generate_sorted_list_of_taxonomy_entries($tagtype, [$lc], {});
+my @category_entries = ProductOpener::TaxonomySuggestions::generate_sorted_list_of_taxonomy_entries($tagtype, $lc, {});
 foreach my $i (0 .. $#category_entries) {
-	$worksheet_categories->write($i, 0, $category_entries[$i]);
+    my $category_entry = $category_entries[$i];
+    
+    $category_entry =~ s/^[a-z]{2}://;
```

Comment:
Hi @TheSussex , when you have a category id like "en:fresh-onions" , it's better to use the function display_taxonomy_tag($target_lc, $tagtype, $canon_tagid); from Tags.pm to display the tag name translated in the language.

That way, if you generate the spreadsheet in French for instance, you will get the French category names:
https://fr.pro.openfoodfacts.org/cgi/generate_sample_import_file.pl

Reply:
You can get the requested language with the $lc variable, and $tagtype is "categories"